exporter/bmchelixexporter/exporter.go

```diff
@@ -0,0 +1,74 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package bmchelixexporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/bmchelixexporter"
+
+import (
+	"context"
+	"errors"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/exporter"
+	"go.opentelemetry.io/collector/pdata/pmetric"
+	"go.uber.org/zap"
+)
+
+// BmcHelixExporter is responsible for exporting metrics to BMC Helix
+type BmcHelixExporter struct {
```

Comment:
I would suggest not making this package in the root package, and by go naming convention it should be `BMCHelixExporter` (all acronyms get capitalised since they are new words).

Conversely, the name is also a double up on the package, perhaps having it just simply as `producer` would be enough to avoid any naming collisions with packages, and adhering to go naming conventions.

Reply:
Thanks for the feedback! That makes sense. I'll rename `BmcHelixExporter` to `metricsExporter` since the package is already named `bmchelixexporter`, and `metricsExporter` should better reflect its purpose within the package.